docs-kits/kits/Industry Core Kit/Software Development View/page_notifications.mdx

```diff
@@ -0,0 +1,173 @@
+
+| Code | Description | |
+|------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---|
+| 200  | OK | Indicates that the request has succeeded. |
+| 4xx  | ERROR | The notification cannot be processed. |
+| 5xx  | ERROR | The notification cannot be processed. |
+
+A response body MAY also be defined (e.g., see Feedback notification operation in the example above).
+
+These error codes are currently optional as the Catena-X connector does not yet return correct error messages from backend services. These error codes will become mandatory once the connector returns correct error messages from backend services.
+
+#### Connector Asset Structure
+
+For a notification API, exactly one asset in the connector must be defined. All notifications operations of the API must be invoked through this asset. 
+
+- The property `http://purl.org/dc/terms/type` must reference the name of the notification API as defined in the Catena-X taxonomy published under https://w3id.org/catenax/taxonomy.
+
+**Example of an asset definition for a notification API:**
+
+```json
+{
+  "@id": "DigitalTwinEventAPI",
+  "@type": "Asset",
+  "properties": {
+    "dct:type": {
+      "@id": "cx-taxo:DigitalTwinEventAPI"
+    },
+    "cx-common:version": "3.0",
+  },
+  "dataAddress": {},
+  "@context": {
+    "dct": "http://purl.org/dc/terms/",
+    "cx-taxo": "https://w3id.org/catenax/taxonomy#",
+    "cx-common": "https://w3id.org/catenax/ontology/common#"
+  }
+}
+```
+
+#### Versioning
+
+Versioning of the notification API must follow the Catena-X guidelines for versioning. For every new major version of a notification API, a new connector asset must be created. If, e.g., a REST API is used as backend notification receiver, the incoming notifications can be fowarded to the correct REST API version via the corresponding connector asset, as depicted in the following figure:
+
+![Notifications Flow](../assets/notifications_connector_versioning.svg)
+
+The API's version is not only documented in the notification's connector asset (property `https://w3id.org/catenax/ontology/common#version`), but also in the OpenAPI specification as well as in a notification's `context` property.
+
+### Notification API Documentation
+
+A use case defining a notification API as part of its standard must document the following artifacts:
+- All notification data formats must be documented either as separate JSON schema or (also in JSON schema) as part of the notification API's OpenAPI specification.
+- The notification API must be documented with an OpenAPI specification that must be published on the Tractus-X Swagger space.
+- For a standardized use case, the name of the notification API (used in property `http://purl.org/dc/terms/type`) must be defined in the Catena-X taxonomy published under https://w3id.org/catenax/taxonomy.
```

Comment:
This is a duplicate sentence and is also mentioned above. I would suggest removing one of them.